app/components/e-info-label.md

```diff
@@ -1,35 +1,35 @@
 ### Promotion
 
-```
-<e-info-label :price-type="1"/>
+```vue
+<e-info-label :price-type=1 />
 ```
 
 ### Promotion with priceTypeEndDate
 
-```
-<e-info-label :price-type="1" price-type-end-date="1.1.1970"/>
+```vue
+<e-info-label :price-type=1 price-type-end-date="1.1.1970"/>
 ```
 
 ### New
 
-```
-<e-info-label :price-type="2"/>
+```vue
+<e-info-label :price-type=2 />
 ```
 
 ### Discount
 
-```
-<e-info-label :price-type="3" price-type-end-date="1.1.1970"/>
+```vue
+<e-info-label :price-type=3 price-type-end-date="1.1.1970"/>
```

Comment:
- [ ] This is not valid in Vue/HTML. If you want to make sure `1` is a string and not a number you have to write quotes inside quotes: `:price-type="'1'"`